psalm.xml

```diff
@@ -82,6 +82,12 @@
             </errorLevel>
         </DeprecatedMethod>
 
+        <ImplementedReturnTypeMismatch>
```

Comment:
@jakubtobiasz How do you know the Psalm level?

Reply:
Idk, I don't like this categorization. I'm wondering about PR removing it, and sorting everything alphabetically. But before I have to get feedback from more people.

I have to admit I don't care about it in my PRs 🤷🏼‍♂️.